lib/css/components/sidebar-widgets.less

```diff
@@ -0,0 +1,245 @@
+.s-sidebarwidget {
+    --_sw-content-px: calc(var(--su16) - var(--su1)); // subtract 1px for border
+    --_sw-content-spacing-inner: var(--su12); // the spacing between two adjacent simple items
+    --_sw-content-spacing-outer: var(--su16); // the spacing at the start/end of a group of simple items, as well as between a complex item and its separator line
+    --_sw-bc: var(--bc-medium);
+    --_sw-content-bc: var(--bc-light);
+    --_sw-header-bg: var(--black-025);
+
+    & &--header {
+        &.s-sidebarwidget__small-bold-text {
+            font-size: var(--fs-caption);
+            font-weight: bold;
+            .s-sidebarwidget--action {
+                font-weight: normal;
+                line-height: calc(1.3 * var(--fs-caption));  // line-height should be the same as in the outside element, so the header and action baselines line up
+            }
```

Comment:
**Another ordering nitpick**: To match the ordering standard in the ADR, I'd suggest placing the style rules for a given element to come at the end of it's block of styles. It might seem weird at first, but I've been doing it that way to ensure that any redefinition of a given PPCP happens before the style is declared.

Reply:
I think I got that rule. This was a genuine miss on my end because of the nesting. Will address it.